parser/src/ast/mod.rs

```diff
@@ -52,10 +55,11 @@ pub enum SourceSection {
     AirDef(Identifier),
     Constant(Constant),
     Trace(Vec<Vec<TraceBinding>>),
+    EvaluatorFunction(EvaluatorFunction),
+    Function(Function),
```

Comment:
The docs above this enum need to be updated, and the order in the docs should be kept consistent with the order of enum variants. Also, I wonder if we should alphabetize things like this in general, now that it's gotten more complicated, rather than trying to follow a mental ordering of how we like to define the AirScript files.